org.zotero.Zotero.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "org.zotero.Zotero",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "zotero",
+    "rename-desktop-file": "zotero.desktop",
+    "rename-icon": "zotero",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=~/.zotero:create",
+        "--filesystem=~/Zotero"
+    ],
+    "modules": [
+        {
+            "name": "zotero",
+            "buildsystem": "simple",
+            "sources": [{
+                "type": "archive",
+                "url": "https://download.zotero.org/client/release/5.0.33/Zotero-5.0.33_linux-x86_64.tar.bz2",
```

Comment:
This appears to have a [package-lock.json](https://github.com/zotero/zotero/blob/master/package-lock.json) upstream so you could use the [flatpak-builder-tools](https://github.com/flatpak/flatpak-builder-tools) to process that and build from source - unless there's any special API keys or similar that would make that difficult.

Reply:
I think it would be quite involving to get node/npm into the flatpak, but I could consider it later.